pages/api/predictions/[id].js

```diff
@@ -1,20 +1,17 @@
+import Replicate from "replicate";
```

Comment:
Coulda put this in a reusable `lib/replicate.js` and used it in both API files, but it's so short that it seems like avoiding the indirection is worth the duplication.